laterpay/asset_sources/scss/objects/_utilities.scss

```diff
@@ -63,11 +63,11 @@
  */
 %clearfix:before,
 %clearfix:after {
-    content: ' '; /* 1 */
-    display: table; /* 2 */
+    content: ' ' !important; /* 1 */
+    display: table !important; /* 2 */
 }
 %clearfix:after {
-    clear: both;
+    clear: both !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces